dev/com.ibm.ws.persistence.ol_fat/test-bundles/com.ibm.ws.persistence.consumer/src/com/ibm/websphere/persistence/test/ConsumerConfig.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.websphere.persistence.test;
+
+import javax.sql.DataSource;
+
+// TODO -- remove
```

Comment:
can this class be removed?

Reply:
Looks like yes, removing it didn't blow anything up.